### DISCUSSION TITLE
Migration Manager --desc doesn't appear to do anything

### DISCUSSION THREAD

**Discussion by theelderbeever** [piccolo-orm/piccolo#427] [Q&A]
Running the command `piccolo migrations new myapp --desc "my_description"` doesn't appear to log the description anywhere. In the migration file there is a constant `DESCRIPTION` that is set to true but the desc isn't located anywhere. It also doesn't have a column in the database. Not sure if this is a bug.

Additionally, a nice to have would be if the autogenerated migration file name and the database could include the description. It makes searching through migrations in the repo a lot easier.
```console
> piccolo migrations new myapp --desc "my_description"
2022-02-09T09-51-38-753759_my_description.py
```
Another option could be a `--suffix` argument to accomplish this. However, I don't personally see these as being separated all that often.

**Comment by dantownsend** [comment]
You make a lot of good points.

The description workaround for now is:

```
> piccolo migrations new myapp --desc="my_description"
```

As for the migration name, I agree that having a timestamp isn't the most user friendly.

You can rename the file. The only thing which can't change is the `ID` value inside.

`piccolo migrations new` should probably have a title argument for this.
